twisted/internet/_sslverify.py

```diff
@@ -1942,3 +1901,61 @@ def fromFile(cls, filePath):
             <twisted.internet.ssl.DiffieHellmanParameters>}
         """
         return cls(filePath)
+
+
+def _setUpNextProtocolMechanisms(context, acceptableProtocols):
```

Comment:
🚗 👋: I feel like there's no point in putting this many words in this function name.  There's absolutely no way to be unambiguous here about the fact the it's NPN or ALPN or both without being insanely wordy.  It's also weird and inconsistent that the function *name* says "next protocol" and "mechanisms" but the *argument* says acceptable "protocols".

So, having spilled far too many bytes on that explanation: how about `_setAcceptableProtocols(context, acceptableProtocols)`?